cluster/gce/util.sh

```diff
@@ -828,17 +864,41 @@ function kube-push {
   get-bearer-token
 
   # Make sure we have the tar files staged on Google Storage
-  find-release-tars
-  upload-server-tars
+  tars_from_version ${1-}
+}
 
+# Upgrade a kubernetes master
+function upgrade-master {
```

Comment:
cluster/gce/upgrade.sh also includes a function called upgrade-master and includes this file. Does it still work? What happens if you define the same function twice in bash?